PCL_RegionGrowing/scripts/save_as_npy.py

```diff
@@ -0,0 +1,18 @@
+import os
+import numpy as np
+
+
+def convert_txt_files_to_npy_format():
+    rootdir = "../output"
+
+    for subdir, _, files in os.walk(rootdir):
+        for file in files:
+            file_path = os.path.join(subdir, file)
+            labels_array = []
+            with open(file_path, "r") as file:
+                lines = file.readlines()
+                for line in lines:
+                    labels_array.append(int(line))
```

Comment:
use `np.loadtxt()`

Reply:
Also, such txt files are tmp ones, don't use output for them (if you will have time fix this, otherwise dont' touch)